apps/voting/app/src/components/VoteSuccess.js

```diff
@@ -19,7 +19,8 @@ const VoteSuccess = ({ settings, vote, ...props }) => (
             <Icon />
           </IconWrapper>
           <StatusLabel>
-            {success ? 'Vote will pass' : 'Vote won’t pass'}
+            Current outcome:
+            {success ? ' Yes' : ' No'}
```

Comment:
I don't think a simple "Yes" or "No" is going to tell a user what's going on here:

![Screenshot_2019-04-15 Aragon(1)](https://user-images.githubusercontent.com/9424721/56175929-20c15480-5fae-11e9-8321-bb2270919065.png)

Reply:
I think Vote won't pass or Vote will pass infers that the vote will take one course or another, without explicitly saying that it's the current state, and not final

Reply:
I see, it is adding a "Current outcome" header followed by Yes or No. Ok.